beacon-chain/slasher/detect_attestations.go

```diff
@@ -193,15 +193,31 @@ func (s *Service) checkDoubleVotes(
 			// This is a double vote.
 			doubleVotesTotal.Inc()
 
-			existing, ok := existingAttWrapper.IndexedAttestation.(*ethpb.IndexedAttestation)
+			existingWrapper, ok := existingAttWrapper.IndexedAttestation.(*slashertypes.WrappedIndexedAtt)
+			if !ok {
+				return nil, fmt.Errorf(
+					"existing attestation has wrong type (expected %T, got %T)",
+					&slashertypes.WrappedIndexedAtt{},
+					existingAttWrapper.IndexedAttestation,
+				)
+			}
+			existing, ok := existingWrapper.IndexedAtt.(*ethpb.IndexedAttestation)
 			if !ok {
 				return nil, fmt.Errorf(
 					"existing attestation has wrong type (expected %T, got %T)",
 					&ethpb.IndexedAttestation{},
 					existingAttWrapper.IndexedAttestation,
```

Comment:
Since we do a type assertion on `existingWrapper.IndexedAtt.(*ethpb.IndexedAttestation)` shouldn't this match?
```suggestion
			existing, ok := existingWrapper.IndexedAtt.(*ethpb.IndexedAttestation)
			if !ok {
				return nil, fmt.Errorf(
					"existing attestation has wrong type (expected %T, got %T)",
					&ethpb.IndexedAttestation{},
					existingWrapper.IndexedAtt,
```